src/css/style.css

```diff
@@ -48,4 +58,327 @@ body {
+        -100px 100px 0 #26517f;
+    position: relative;
+}
+
+.cui-long-shadow:before {
+    content: " ";
+    background: #26517f;
+    width: calc(100% + 38px);
+    height: 500px;
+    -webkit-transform: skew(135deg);
+    transform: skew(135deg);
+    -webkit-transform-origin: top right;
+    transform-origin: top right;
+    position: absolute;
+    top: 100%;
+    left: -59px;
+}
+
+.cui-form-group {
+    margin-bottom: 18px;
+}
+
+.cui-input {
+    height: 54px;
+    border: 4px solid #A4D4D1;
+    border-radius: 0;
+}
+
+.cui-input:active,
+.cui-input:focus {
+    border-color: #48A4AE;
+}
+
+.cui-button {
+    background: #c9302c;
+    height: 54px;
+    border: 4px solid #ac2925;
+    border-radius: 0;
+    text-transform: uppercase;
+    font-family: 'Enriqueta';
+    letter-spacing: 0.05em;
+    text-shadow:
+        -1px 1px 0 #ac2925,
+        -2px 2px 0 #ac2925;
+}
+
+.cui-button:hover {
+    background: #ac2925;
+    border-color: #ac2925;
+}
+
+.loading-img + .cui-button {
+    color: transparent;
+    text-shadow: none;
+}
+
+.loading-img.hidden + .cui-button {
+    color: #fff;
+    text-shadow:
+        -1px 1px 0 #ac2925,
+        -2px 2px 0 #ac2925;
+}
+
+.cui-text-rule {
+    margin: 0 0 16px 0;
+    font-family: 'Enriqueta';
+    letter-spacing: 0.1em;
+    font-style: italic;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-align: center;
+    align-items: center;
+}
+
+.cui-text-rule:before,
+.cui-text-rule:after {
+    content: " ";
+    background: #fff;
+    height: 1px;
+    margin: 0 15px 0 0;
+    display: block;
+    -webkit-box-flex: 2;
+    flex-grow: 2;
+}
+
+.cui-text-rule:after {
+    margin: 0 0 0 15px;
+}
+
+.cui-panels h3 {
+    margin: 36px 0;
+}
+
+.cui-panels .panel {
+    background: #f5f5f5;
+    margin: 0 0 36px 0;
+    border: 4px solid #ddd;
+    box-shadow: none;
+    border-radius: 0;
+}
+
+.cui-panels .legislator-name {
+    font-family: 'Enriqueta';
+}
+
+.cui-panels .thumbnail {
+    border-radius: 0;
+}
+
+@media (min-width: 768px) {
+
+    .cui-logo {
+        width: 250px;
+    }
+
+    .cui-jumbotron-left {
+        padding-top: 54px;
+    }
+
+    .cui-jumbotron-right {
+        padding-top: 54px;
+        padding-bottom: 72px;
+    }
+
+    .cui-jumbotron-header {
+        font-size: 72px;
+    }
+
+    .cui-long-shadow:before {
+        left: -67px;
+    }
+}
+
+@media (min-width: 992px) {
+
+    .cui-row-flex {
+        display: -webkit-box;
+        display: flex;
+        -webkit-box-align: stretch;
+        align-items: stretch;
+    }
+
+    .cui-jumbotron-left {
+        border: none;
+        padding-right: 36px;
+        padding-bottom: 54px;
+    }
+
+    .cui-jumbotron-right {
+        padding-top: 72px;
+        padding-left: 36px;
+    }
+
+    .cui-panels:nth-child(1) {
+        padding-right: 36px;
+    }
+
+    .cui-panels:nth-child(2) {
+        padding-left: 36px;
+    }
+
+    .cui-panels .panel {
+        margin: 0 0 54px 0;
+    }
```

Comment:
do you mind moving all of these styles to cui.css in styles folder and just including that in the index.html? 

It seems to be a good organization since all of them have .cui-* in the class declaration